docs/Compounds.md

```diff
@@ -0,0 +1,12 @@
+# Compounds page
```

Comment:
This page is formatted differently from Edit_Page in its use of sub-headings. I like Edit_page better. Please update this one to emulate that.